scripts/proto_splitter/protoBatchEdit.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python3
+
+# Copyright 1996-2021 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author Simon Steinmann (https://github.com/Simon-Steinmann)
+#
+
+
+# Automatic PROTO to multi-file PROTO trimesh extractor
```

Comment:
You should at least update this initial explanation (and instructions) as it is seems the script is not only used for trimesh extraction.
Otherwise this script will never be used again.

Reply:
Is there interest in the script? What it does is adding a line to the header, and to the baseNode to all .proto files in a directory tree. With some filter criteria added to it.

Reply:
Can this tool as-is be used to apply the changes in PR to a batch of PROTO files?
If yes, it may be useful in the future but only if properly documented.
If not and only work as trimesh extractor in the committed version then this PR is not the correct place where to add it.